anago

```diff
@@ -970,7 +970,10 @@ announce () {
     logecho "$WARNING: Email cannot be sent from GCB.  Execute the following" \
             "after a completed release to send email notification:"
     logecho
-    logecho "$ release-notify $RELEASE_VERSION_PRIME"
+    local nomock_flag
+    ((FLAGS_nomock)) && nomock_flag="--nomock"
+    logecho "$ release-notify $nomock_flag $RELEASE_VERSION_PRIME"
```

Comment:
Use $PROG instead of release-notify

Reply:
$PROG would be anago here, right? The message should say `release-notify` not anago.

Reply:
oops, yes.  Thanks for paying attention.

/lgtm